sites/automationworld/redirect-handler.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  check: (from) => {
```

Comment:
Export this as a function, not an object with a function property... i.e.:

```js
module.exports = ({ from }) => {
   // your logic here
};
```

**Note**: I'm also passing `from` as an destructured object property. This way, you can pass the function straight through. See example below.

This change applies to all files

Reply:
I'll update that!